lib/core/include/irods/rodsErrorTable.h

```diff
@@ -240,6 +240,7 @@ NEW_ERROR(NOT_A_DATA_OBJECT,                           -171000)
 NEW_ERROR(JSON_VALIDATION_ERROR,                       -172000)
 NEW_ERROR(DATABASE_TYPE_NOT_SUPPORTED,                 -173000)
 NEW_ERROR(RESOLUTION_ERROR,                            -174000)
+NEW_ERROR(NO_CATALOG_SERVICE_PROVIDER,                 -175000)
```

Comment:
Also, this can be removed. I will include that with the #s

Reply:
Are we keeping this?

Reply:
Nevermind. I misread your comment.

Reply:
Done.